www/content/tutorial.md

```diff
@@ -110,58 +110,32 @@ Roc will respect [order of operations](https://en.wikipedia.org/wiki/Order_of_op
 
 Let's try calling a function:
 
-<pre><samp><span class="repl-prompt">Str.concat "Hi " "there."</span>
+<pre><samp><span class="repl-prompt">Str.concat("Hi ", "there.")</span>
 
 <span class="literal">"Hi there."</span> <span class="colon">:</span> Str
 </samp></pre>
 
 Here we're calling the `Str.concat` function and passing two arguments: the string `"Hi "` and the string `"there."`. This _concatenates_ the two strings together (that is, it puts one after the other) and returns the resulting combined string of `"Hi there."`.
 
-Note that in Roc, we don't need parentheses or commas to call functions. We don't write `Str.concat("Hi ", "there.")` but rather `Str.concat "Hi " "there."`.
-
-That said, just like in the arithmetic example above, we can use parentheses to specify how nested function calls should work. For example, we could write this:
-
-<pre><samp><span class="repl-prompt">Str.concat "Birds: " (Num.toStr 42)</span>
-
-<span class="literal">"Birds: 42"</span> <span class="colon">:</span> Str
-</samp></pre>
-
-This calls `Num.toStr` on the number `42`, which converts it into the string `"42"`, and then passes that string as the second argument to `Str.concat`.
-
-The parentheses are important here to specify how the function calls nest. Try removing them, and see what happens:
-
-<pre><samp><span class="repl-prompt">Str.concat "Birds: " Num.toStr 42</span>
-
-<span class="repl-err">&lt;error&gt;</span>
-</samp></pre>
-
-The error tells us that we've given `Str.concat` too many arguments. Indeed we have! We've passed it three arguments:
-
-1.  The string `"Birds"`
-2.  The function `Num.toStr`
-3.  The number `42`
-
-That's not what we intended to do. Putting parentheses around the `Num.toStr 42` call clarifies that we want it to be evaluated as its own expression, rather than being two arguments to `Str.concat`.
-
-Both the `Str.concat` function and the `Num.toStr` function have a dot in their names. In `Str.concat`, `Str` is the name of a _module_, and `concat` is the name of a function inside that module. Similarly, `Num` is a module, and `toStr` is a function inside that module.
+Both the `Str.concat` function has a dot in their names. In `Str.concat`, `Str` is the name of a _module_, and `concat` is the name of a function inside that module.
```

Comment:
This line still mentions "Both" but `Num.toStr` was removed.

Reply:
Done